modules/ROOT/nav.adoc

```diff
@@ -34,4 +35,4 @@
  ** xref:config-add-proxy-whitelist.adoc[Add Whitelist Addresses for API Platform Proxies]
  ** xref:demo-ldap-server.adoc[Configure the Demo LDAP Server]
  ** xref:ext-analytics-elk.adoc[Analyze Business and API Data using ELK]
-* xref:upgrade.adoc[Migrating Anypoint Platform Private Cloud]
\ No newline at end of file
+* xref:upgrade.adoc[Migrating Anypoint Platform Private Cloud]
```

Comment:
Is this part of this PR?

Reply:
May have slimed this in for another ticket - will have to verify.